core/data/tests/struct_decorator/input.rs

```diff
@@ -15,7 +15,7 @@ pub struct BestHockeyTeams2 {
     Lies: String,
 }
 
-#[typeshare(kotlin = "redacted_to_string")]
+#[typeshare(kotlin = "Redacted")]
```

Comment:
I think the other languages can also have redacted output. It may be better for this to be parsed as `#[typeshare(redacted)]` allowing each language formatter to choose how the redacted type should be implemented.